### PR TITLE
Feature/inflatable rigid bodies

### PR DESCRIPTION
**Pull request by InfiniteLee**
Duck interactables updated to use `shape` components.
Most of the work to support this is in: https://github.com/donmccurdy/aframe-physics-system/pull/90
Note: the interactables will be moved into dynamically loaded gltf bundles soon. Additionally, the environments will make use of `shape` components soon as well.

### CODE REVIEW

File: src/assets/interactables/duck/gltf/DuckyMesh.gltf.meta [hunk 1]
```diff
@@ -0,0 +1 @@
...erView":0,"byteOffset":0},{"componentType":5126,"type":"VEC3","count":477,"bufferView":1,"byteOffset":0,"min":[-0.0587991699576378,0.0129461474716663,-0.0740185976028442],"max":[0.0618169121444225,0.173104390501976,0.104356855154037]},{"componentType":5126,"type":"VEC3","count":477,"bufferView":2,"byteOffset":0},{"componentType":5126,"type":"VEC2","count":477,"bufferView":3,"byteOffset":0}],"images":[{"name":"DuckyMesh.fbm/Ducky.jpg","uri":"DuckyMesh.fbm/Ducky.jpg"}],"samplers":[{}],"textures":[{"name":"file4","sampler":0,"source":0}],"materials":[{"name":"lambert2","alphaMode":"OPAQUE","extras":{"fromFBX":{"shadingModel":"Lambert","isTruePBR":false}},"pbrMetallicRoughness":{"baseColorTexture":{"index":0,"texCoord":0},"baseColorFactor":[0.800000011920929,0.800000011920929,0.800000011920929,1],"metallicFactor":0.200000002980232,"roughnessFactor":0.800000011920929}}],"meshes":[{"name":"Ducky","primitives":[{"material":0,"mode":4,"attributes":{"NORMAL":2,"POSITION":1,"TEXCOORD_0":3},"indices":0}]}],"nodes":[{"name":"RootNode","translation":[0,0,0],"rotation":[0,0,0,1],"scale":[1,1,1],"children":[1]},{"name":"Ducky","translation":[0,0,0],"rotation":[0,0,0,1],"scale":[1,1,1],"mesh":0}]}
```
Comment (netpro2k) [inline src/assets/interactables/duck/gltf/DuckyMesh.gltf.meta]: Whats this .meta file?

File: src/assets/interactables/duck/gltf/DuckyMesh.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          "orientation": {
+            "x": 0,
+            "y": 0,
+            "z": 0,
+            "w": 1
+          }
+        },
+        {
+          "shape": "cylinder",
+          "radiusTop": 0.02,
```
Comment (netpro2k) [inline src/assets/interactables/duck/gltf/DuckyMesh.json]: Weird that Cannon cylinders have both a top and bottom radius.. but I guess that lets you make some interesting shapes.

File: src/components/super-cursor.js [hunk 3]
```diff
@@ -103,6 +103,18 @@ AFRAME.registerComponent("super-cursor", {
     }
   },
 
+  _isInteractable: function(el) {
+    if (el.className === "interactable") {
+      return true;
+    }
+
+    if (el.parentNode && el.parentNode != el.sceneEl) {
```
Comment (netpro2k) [inline src/components/super-cursor.js]: Hmm, walking up the tree like this every frame is a bit concerning. Also I would inline this recursion. Shouldn't the raycaster be handling this filtering for you anyway?

File: src/hub.html [hunk 4]
```diff
@@ -105,11 +105,11 @@
 
             <template id="interactable-template">
                 <a-entity
-                    gltf-model-plus="src: #interactable-duck"
+                    gltf-model-plus="src: #interactable-duck; inflate: true;"
                     scale="2 2 2"
                     class="interactable" 
                     super-networked-interactable="counter: #counter; mass: 5;"
-                    body="type: dynamic; mass: 5; shape: box;"
+                    body="type: dynamic; shape: none; mass: 5;"
```
Comment (netpro2k) [inline src/hub.html]: ooh lighter ducks :duck:

File: src/assets/interactables/duck/gltf/DuckyMesh.gltf.meta (hunk 1 above)
Comment (InfiniteLee) [reply src/assets/interactables/duck/gltf/DuckyMesh.gltf.meta]: I have no idea. It got generated when I exported the .gltf model. Are those not just part of the gltf? I can delete it.

File: src/assets/interactables/duck/gltf/DuckyMesh.json (hunk 2 above)
Comment (InfiniteLee) [reply src/assets/interactables/duck/gltf/DuckyMesh.json]: Only way to get a cone

**Pull request merged**